test/com/ableton/PythonPackageVersionReaderTest.groovy

```diff
@@ -0,0 +1,85 @@
+package com.ableton
+
+import static org.junit.Assert.assertEquals
+import static org.junit.Assert.assertNotNull
+import static org.junit.Assert.assertTrue
+
+import com.lesfurets.jenkins.unit.BasePipelineTest
+import org.junit.Before
+import org.junit.Test
+
+
+/**
+ * Tests for the PythonPackageVersionReader class.
+ */
+class PythonPackageVersionReaderTest extends BasePipelineTest {
+  @SuppressWarnings('FieldTypeRequired')
+  def script
+
+  @Override
+  @Before
+  void setUp() throws Exception {
+    super.setUp()
+
+    this.script = loadScript('test/resources/EmptyPipeline.groovy')
+    assertNotNull(script)
```

Comment:
I don't thing this check is really necessary (though it isn't hurting anything either).

Reply:
Fixed as of 2437eb2.